libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -124,10 +124,26 @@ singleton AP_Notify alias notify
 singleton AP_Notify method play_tune void string
 singleton AP_Notify method handle_rgb void uint8_t 0 UINT8_MAX uint8_t 0 UINT8_MAX uint8_t 0 UINT8_MAX uint8_t 0 UINT8_MAX
 
+include AP_Proximity/AP_Proximity.h
+
+singleton AP_Proximity alias proximity
+singleton AP_Proximity method num_sensors uint8_t
+singleton AP_Proximity method get_object_count uint8_t
+singleton AP_Proximity method get_closest_object boolean float'Null float'Null
+singleton AP_Proximity method get_object_angle_and_distance boolean uint8_t 0 UINT8_MAX float'Null float'Null
+
 include AP_RangeFinder/AP_RangeFinder.h
 
 singleton RangeFinder alias rangefinder
 singleton RangeFinder method num_sensors uint8_t
+singleton RangeFinder method has_orientation boolean Rotation'enum ROTATION_NONE ROTATION_ROLL_90_PITCH_315
+singleton RangeFinder method distance_cm_orient uint16_t Rotation'enum ROTATION_NONE ROTATION_ROLL_90_PITCH_315
+singleton RangeFinder method voltage_mv_orient uint16_t Rotation'enum ROTATION_NONE ROTATION_ROLL_90_PITCH_315
```

Comment:
Can we remove this? Nothing in the internal codebase uses it, and it's really hard to see how it's a good method to be exposing here from the rangefinder. If you don't have a strong use case for it I think we should be nuking the method and the associated helpers out of the code base.

Reply:
We should probably remove the while method too....

Reply:
Yeah, I want to kill the C++ part of it, but that doesn't have to be in this PR.

Reply:
I need distance_cm_orient. I agree to delete it.